common.sh

```diff
@@ -464,7 +465,7 @@ solution_file() {
     # solution file will be created in module directory
     # :param message: Message - string of list of strings
     #
-    echo "$1" >> "$SOLUTION_FILE"
+    echo "$@" >> "$SOLUTION_FILE"
```

Comment:
The exit_failed accepts no argument, as I'm looking in the code. And the other functions that accept a message seem to be fixed this way too.

BTW, is there a any reason **why there's *exit_failed* in python API and *exit_fail~~ed~~* in bash API**?